### DISCUSSION TITLE
Welcome to k3s Discussions!

### DISCUSSION THREAD

**Discussion by cjellick** [k3s-io/k3s#2968] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, please feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nunix** [comment]
Great initiative and the Corsair is pleased to be ~~stalking~~ following the great K3s community.

As a short intro: I do everything from WSL2 😇
So if there is any questions around it and how to use K3s on it, tag me and I will appear.

**Reply by cjellick** [reply]
Hi @nunix. Thanks! We definitely have a lot of interest in k3s + wsl2 going forward.

**Comment by Brice187** [comment]
When will k3s switch to traefik 2?

**Reply by cjellick** [reply]
PR is here: https://github.com/k3s-io/k3s/pull/2910
Plan is to have it land in the next minor release, so 1.21, which means april.

**Comment by saiyam1814** [comment]
Nice to have Discussion, and 👋 everyone

**Comment by Brice187** [comment]
I use to install k3s via ansible. Is it possible to set .kubeconfig as installation parameter? So I can store it in my secrets and bootstrap my clusters without manually obtaining .kubeconfig after ansible rollout
